PWGJE/macros/AddTaskJetCoreEmcal.C

```diff
@@ -0,0 +1,76 @@
+AliAnalysisTaskJetCoreEmcal* AddTaskJetCoreEmcal(
+								 const char *njetsBase,
+						     const Double_t R,
+								 const char *type,
+								 Int_t jetShapeType = AliAnalysisTaskJetCoreEmcal::kData,
+								 Float_t kTTminr=8,
+								 Float_t kTTmaxr=9,
+								 Float_t kTTmins=20,
+								 Float_t kTTmaxs=50,
+								 const char *listName = ""
+                 ) {
+  
+  AliAnalysisManager *mgr = AliAnalysisManager::GetAnalysisManager();
+  if (!mgr)
+    {
+      Error("AddTaskEmcalQGTagging","No analysis manager found.");
+      return 0;
+    }
+  Bool_t ismc=kFALSE;
+  ismc = (mgr->GetMCtruthEventHandler())?kTRUE:kFALSE;
+
+  // Check the analysis type using the event handlers connected to the analysis manager.
+  //==============================================================================
+  if (!mgr->GetInputEventHandler())
+    {
+      ::Error("AddTaskEmcalQGTagging", "This task requires an input event handler");
+      return NULL;
+    }
+
+  TString wagonName1 = Form("JetCore_%s_TC",njetsBase);
+  TString wagonName2 = Form("JetCore_%s_TCTree",njetsBase);
+  //Configure jet tagger task
+  AliAnalysisTaskJetCoreEmcal *task = new AliAnalysisTaskJetCoreEmcal(wagonName1.Data());
+	task->SetTTLowRef(kTTminr);
+	task->SetTTUpRef(kTTmaxr);
+	task->SetTTLowSig(kTTmins);
+	task->SetTTUpSig(kTTmaxs);
+
+	TString name = "JetCore";
+	TString trackName = "tracks";
+	TString clusName = "caloClusters";
+  if (trackName == "mcparticles") {
+    task->AddMCParticleContainer(trackName);
+  }
+  else if (trackName == "tracks" || trackName == "Tracks") {
+    task->AddTrackContainer(trackName);
+  }
+  else if (!trackName.IsNull()) {
+    task->AddParticleContainer(trackName);
+  }
+  task->AddClusterContainer(clusName);
+
+  //-------------------------------------------------------
+  // Final settings, pass to manager and set the containers
+  //-------------------------------------------------------
+
+  mgr->AddTask(task);
+
+  // Create containers for input/output
+  AliAnalysisDataContainer *cinput1  = mgr->GetCommonInputContainer()  ;
+  TString contname(name);
+	contname += Form("_%s",TString(listName).Data());
+	contname += Form("_%02d",Int_t(R*10+0.001));
+  contname += "_histos";
+  AliAnalysisDataContainer *coutput1 = mgr->CreateContainer(contname.Data(),
+      TList::Class(),AliAnalysisManager::kOutputContainer,
+      Form("%s", AliAnalysisManager::GetCommonFileName()));
+  mgr->ConnectInput  (task, 0,  cinput1 );
+  mgr->ConnectOutput (task, 1, coutput1 );
+	cout<<"emcal add task settings - return;"<<endl;
```

Comment:
Needs to be `std::cout` for ROOT 6 compatibility. Generally good to use AliDebugStream(int), AliInfoStream(), etc.

Reply:
This is in a macro, AliDebug/Info/WarningStream won't work here (required to be called within non-static member functions).  As long it is in the user macro missing std:: - though not recommendable - can be tolerated. However in the compiled code (see above) this must be fixed because it will break compilation

Reply:
thanks for suggestions - this was leftover from testing and I don't need this output. I have removed in latest push

Reply:
@mfasDa - Yeah, you're right, I wasn't being super precise. One could use `AliDebugGeneralStream(...)` for such situations, but in any case, it's irrelevant here